test/endtoend/operator/101_initial_cluster.yaml

```diff
@@ -8,11 +8,11 @@ metadata:
   name: example
 spec:
   images:
-    vtctld: vitess/lite:v19.0.4
-    vtgate: vitess/lite:v19.0.4
-    vttablet: vitess/lite:v19.0.4
-    vtorc: vitess/lite:v19.0.4
-    vtbackup: vitess/lite:v19.0.4
+    vtctld: vitess/lite:v20.0.0-rc1
+    vtgate: vitess/lite:v20.0.0-rc1
+    vttablet: vitess/lite:v20.0.0-rc1
+    vtorc: vitess/lite:v20.0.0-rc1
+    vtbackup: vitess/lite:v20.0.0-rc1
```

Comment:
This is the image that will be used as the version `N-1` to start the upgrade test. Later in the test workflow, we apply another file that will use `:latest`.

Ensuring that using the same vtop version but going from vitess `v20.0.x` to `main` works.